styles/about.css

```diff
@@ -0,0 +1,85 @@
+/* About section */
+.about-section {
+  padding: 7.125rem 1.5rem;
+  width: 100vw;
+  border-top-right-radius: 5.0625rem;
+  background-color: #fff;
+}
```

Comment:
- [ ] This section should be aligned using a `flexbox`, please fix that.😁

Reply:
- I have made made the changes , kindly let me know if they went through 😄

Reply:
@mirouhml Thank you for your good tips

Reply:
@michael-duke Good job on making the changes, please submit again through the dashboard.